src/Components/TagPickerComponent/.lintstagedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "*.ts": "eslint --fix",
```

Comment:
Not 100% but I think this can be formatted  "*.{ts,tsx}":"" to avoid duplicating the command